### PR TITLE
Close menu on navigation

### PR DESCRIPTION
**Pull request by lukebrooker**
Could do this by attaching a root event handler… but didn't want to deal with that 🤷‍♂️

### CODE REVIEW

File: source/components/Header/index.js [hunk 1]
```diff
@@ -34,22 +34,22 @@ const Header = ({
...
-          <NavLink to='/resources'>Resources</NavLink>
-          <NavLink title='Frequently asked questions' to='/faqs'>FAQs</NavLink>
-          <NavLink title='Terms and conditions' to='/terms-conditions'>T &amp; C's</NavLink>
+          <NavLink onClick={onToggleOff} to='/fundraising-ideas'>Fundraising Ideas</NavLink>
+          <NavLink onClick={onToggleOff} to='/tips'>Tips</NavLink>
+          <NavLink onClick={onToggleOff} to='/resources'>Resources</NavLink>
+          <NavLink onClick={onToggleOff} title='Frequently asked questions' to='/faqs'>FAQs</NavLink>
+          <NavLink onClick={onToggleOff} title='Terms and conditions' to='/terms-conditions'>T &amp; C's</NavLink>
           {user.token ? (
             <NavLink title='Logout' onClick={clearSession}>Log out</NavLink>
```
Comment (kizza) [inline source/components/Header/index.js]: Do you want this guy to `onToggleOff && clearSession`... or perhaps not as required as the other nav links?

**Comment by rowanhogan** [comment]
Is this good to go?

File: source/components/Header/index.js (hunk 1 above)
Comment (lukebrooker) [reply source/components/Header/index.js]: Na all good as it reloads the page

**Comment by lukebrooker** [comment]
GTG!!

**Pull request merged**